vdso/makefile

```diff
@@ -1,21 +1,25 @@
+arch = $(shell uname -m)
```

Comment:
I think this is incorrect. It would not work with cross-compilation as the target would be native.
i.e. if you are on x86 and are cross-compiling to arm, this would cause it to compile to arm??
Please, prefer to use environment exported by the build environment.
e.g. CARGO_CFG_TARGET_ARCH

Reply:
There is no such env here because vdso is not build by cargo, and I think the makefile in Quark also detect arch by this, and we can consider support cross-compilation later in a unified way.

Reply:
we will add cross compile features after, as you suggested.